wagtail/users/tests/test_admin_views.py

```diff
@@ -1791,6 +1826,23 @@ def test_group_edit(self):
         group = Group.objects.get(pk=self.test_group.pk)
         self.assertEqual(group.name, "test group edited")
 
+        # On next load of the edit view,
+        # should render the status panel with the last updated time
+        response = self.get()
+        self.assertContains(response, "test group edited")
+        soup = self.get_soup(response.content)
+        status_panel = soup.select_one('[data-side-panel="status"]')
+        self.assertIsNotNone(status_panel)
+        last_updated = status_panel.select_one(".w-help-text")
+        self.assertIsNotNone(last_updated)
+        self.assertRegex(
+            last_updated.get_text(strip=True),
+            f"[0-9][0-9]:[0-9][0-9] by {self.user.get_username()}",
```

Comment:
Note to self: This test is currently failing (as in, 17:00 BST on 8 Apr) since it's outputting "Just now" instead of an actual timestamp. This seems to be due to https://github.com/wagtail/wagtail/blob/d3da555a31d8a1ccce57281c8839ba27a2546a19/wagtail/admin/templatetags/wagtailadmin_tags.py#L783 mixing naive and timezone-aware datestamps and returning false, leading it to reach the special case for "just now" further down which would logically never occur if the date comparison was correct. 🤦

Reply:
Ah yep, I hit this issue locally too, but the fix was so unrelated to this PR so I was planning to create a separate issue or wait until this is somehow reproduced on the CI 😅 

The naive vs aware datetime issue can be fixed by moving the conversion inside the if block to be done first, but yep the “just now” logic would never happen, so that needs fixing too.